site/content/partner-program-api/index.md

```diff
@@ -1178,5 +1573,123 @@ curl --request GET \
     ]
 }
 ```
+
+`GET /v2/partners/experts/history` - get historical stats for Expert profile views (up to last 12 months)
+
+#### Headers
+* `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
+
+#### Response
+* `200 - OK`
 * `401 - Unauthorized` - missing or incorrect authorization header
 * `403 - Forbidden` - Partner is not an Expert
+
+
+# Withdrawals
+
+## Request Withdrawal
+
+>Response example:
+
+```json
+{
+    "request_timestamp": "2018-03-06 12:24:42",
+    "action_timestamp": "",
+    "status": "pending",
+    "amount": 1520,
+    "invoice_id": "2018/2/xyz"
+}
+```
+
+`POST /v2/partners/affiliates/withdrawals` - create withdrawal request
+
+#### Headers
+* `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
+
+#### Response
+* `201 - Created`
+* `401 - Unauthorized` - missing or incorrect authorization header
+* `403 - Forbidden` - not enough money to withdraw (min. $50)
+* `409 - Conflict` - billing info not found, please [update billing](#update-billing)
+
+
+## Get Withdrawal List
+
+>Response example:
+
+```json
+{
+    "result": [
+        {
+            "request_timestamp": "2018-03-06 12:24:42",
+            "action_timestamp": "",
+            "status": "pending",
+            "amount": 1520,
+            "invoice_id": "2018/2/xyz",
+            "withdrawal_id": 1
+        },
+        {
+            "request_timestamp": "2018-02-02 11:43:37",
+            "action_timestamp": "2018-02-02 12:04:11",
+            "status": "paid",
+            "amount": 1063.3,
+            "invoice_id": "2018/1/xyz",
+            "withdrawal_id": 2
+        }
+    ]
+}
+```
+
+`GET /v2/partners/affiliates/withdrawals` - get affiliate withdrawals
+
+#### Headers
+* `Authorization` - **required** - `Bearer <YOUR_API_TOKEN>`
+
+#### Query parameters
+* `date_from` - optional - get report for withdrawals from a given date
+* `date_to` - optional - get report for withdrawals to a given date
+
+**Note:** Date format: `YYYY-MM-DD`
+
+#### Response
+* `200 - OK`
+* `400 - Bad Request` - incorrect parameters
+* `401 - Unauthorized` - missing or incorrect authorization header
+
+
+## Get Withdrawal's Invoice
+
+>Response example:
+
+```json
+{
+    "company": "Your Company",
+    "address": "Street",
+    "city": "City",
+    "zip_code": "12345",
+    "country": "Poland",
+    "vat_id": "",
+    "amount": 1500,
+    "paypal": "jane@email.com",
+    "lc_company": "LiveChat, Inc.",
+    "lc_address": "One International Place, Suite 1400",
+    "lc_city": "Boston, MA",
+    "lc_zip_code": "02110",
+    "lc_country": "United States of America"
+}
+```
+
+>Note: We use this endpoint to generate the invoices PDF in our App
```

Comment:
I guess this note is not necessary. Code and description is clear, no need to say about PDF in my opinion.